packages/mds-agency/tests/test.ts

```diff
@@ -564,17 +539,10 @@ describe('Tests API', () => {
       })
   })
 
-  it('resets the cache', done => {
-    request
-      .get('/test/reset')
-      .set('Authorization', AUTH)
-      .expect(200)
-      .end((err, result) => {
-        test.value(result).hasHeader('content-type', APP_JSON)
-        test.string(result.body.result).contains('cache reset done')
-        done(err)
-      })
+  it('resets the cache', async () => {
```

Comment:
Are these tests even still required? If so, it seems like they should be in `mds-cache`, `mds-db`, `mds-stream` as appropriate.